### DISCUSSION TITLE
Outlet 组件我如何向里面子组件传传参数

### DISCUSSION THREAD

**Discussion by juzhiqiang** [umijs/umi#9184] [Q&A]
Outlet 组件我如何向里面子组件传传参数

**Comment by fz6m** [comment]
可以使用 [useOutletContext](https://umijs.org/docs/api/api#useoutletcontext)

FYI :

 - react router : [useOutletContext](https://reactrouter.com/en/main/hooks/use-outlet-context#useoutletcontext)

**Reply by juzhiqiang** [reply]
好的 谢谢
